config/macros.tex

```diff
@@ -387,11 +382,21 @@
 \newcommand{\addressLen}{\ensuremath{\constantstyle{ADDRLEN}}\xspace} % Bit length of an Ethereum address
 \newcommand{\ethWordLen}{\ensuremath{\constantstyle{ETHWORDLEN}}\xspace} % Bit length of an Ethereum word
 \newcommand{\txDefaultGas}{\ensuremath{\constantstyle{DGAS}}\xspace} % Intrinsic gas cost of an Ethereum transaction
+%% The general group and field
+\newcommand{\Curve}{\ensuremath{\curvestyle{Curve}}\xspace} % Chosen curve
+\newcommand{\rCURVE}{\ensuremath{\constantstyle{r}}\xspace} % Prime that defines the scalar field \FFx{\rCURVE} of \Curve
+\newcommand{\fieldBitLen}{\ensuremath{\constantstyle{FIELDLEN}}\xspace} % Bit-length of elements in \FFx{\rCURVE}
+\newcommand{\fieldBitCap}{\ensuremath{\constantstyle{FIELDCAP}}\xspace} % Bit-capacity of elements in \FFx{\rCURVE}
 %% Remarkable groups and fields
-\newcommand{\bnFieldBitLen}{\ensuremath{\constantstyle{BNFIELDLEN}}\xspace} % Bit length of an element of the BN254 scalar field
-\newcommand{\bnFieldBitCap}{\ensuremath{\constantstyle{BNFIELDCAP}}\xspace} % Maximum bit length such that all elements of that bit length are in BN254 scalar field
+\newcommand{\BNCurve}{\ensuremath{\curvestyle{BN\mhyphen{}254}}\xspace}
+\newcommand{\rBN}{\ensuremath{\constantstyle{r_{BN}}}\xspace} % Instance of \r for the scalar field of BN254
+\newcommand{\bnFieldBitLen}{\ensuremath{\constantstyle{FIELDLEN_{BN}}}\xspace} % Instance of \fieldBitLen for the BN254 scalar field
+\newcommand{\bnFieldBitCap}{\ensuremath{\constantstyle{FIELDCAP_{BN}}}\xspace} % Instance of \fieldBitCap for the BN254 scalar field
+\newcommand{\BLSCurve}{\ensuremath{\curvestyle{BLS12\mhyphen{}377}}\xspace}
+\newcommand{\rBLS}{\ensuremath{\constantstyle{r_{BLS}}}\xspace} % Instance of \r for the scalar field of BLS12-377
```

Comment:
Use of abbreviation `BLS` (in macro name and constants) for BLS12-377.

Reply:
Sounds good to me. If we want to use several curves from a same family we can still refine these macros going forward, but for now this is clear enough and not necessary.

Reply:
OK.  Sounds good.